### ISSUE TITLE
Dashboard\Admin\Contacts - User has conditional tile link display

### ISSUE THREAD

**Issue by shon-button** [bcgov/cas-registration#2166]
#### Description:

The "Contacts Management" tile should only appear on the main dashboard under specific circumstances.
[Figma link](https://www.figma.com/file/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?type=design&node-id=7702-4102&mode=design&t=urN0lS2jx1k3p5Ve-4)

#### Acceptance Criteria:

Given I am an authenticated user with the `industry-user` role (i.e., I have been Approved to have `reporter` rights for an Operator)
When I navigate to the /dashboard page
Then the "Contacts" tile displays/does not display in the dashboard? **attn: Nan**

Given I am an authenticated user with the `cas-admin` role
When I navigate to the /dashboard page
Then the "Contacts" tile displays/does not display in the dashboard? **attn: Nan**

Given I am an authenticated user with the `cas-analyst` role
When I navigate to the /dashboard page
Then the "Contacts" tile displays/does not display in the dashboard? **attn: Nan**

#### Development Checklist:
- [ ] Contacts tile renders conditionally based on user roles
- [ ] update E2E tests for each user role
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Dependencies

- Blocked by

**Comment by patriciarussellCAS** [comment]
Moose team - can we review the AC here and refine this ticket?

**Comment by BCerki** [comment]
I believe this all works the way it's supposed to now since we completed the tickets about role access from the miro board. All cas users and all approved industry users should be able to see the contacts tile
